cli/compose/schema/data/config_schema_v3.8.json

```diff
@@ -539,6 +539,12 @@
         },
         "labels": {"$ref": "#/definitions/list_or_dict"},
         "driver": {"type": "string"},
+        "driver_opts": {
```

Comment:
can you squash this with the other schema-changes? or (perhaps cleaner) do a re-generate of the bindata in this commit as well?